coverage.out

```diff
@@ -0,0 +1,351 @@
+mode: set
```

Comment:
Does this file add anything?  I don't see any additions of things that generate it or use it.